packages/seeder/src/seeder.js

```diff
@@ -64,6 +64,7 @@ class Seeder extends EventEmitter {
     await this.store.ready()
 
     this.networker = new Networker(this.store)
+    await this.networker.listen()
```

Comment:
This is necessary for some reason? I understand that `corestore-networker` calls listen automatically for you before `configure` anything.

Reply:
Something is not being triggered (ready state or swarm listen) without peers connected. Thanks for the comments and fixes @dpaez!